reference/forms/types/entity.rst

```diff
@@ -194,39 +194,16 @@ return a ``QueryBuilder``.
 Overridden Options
 ------------------
 
-choice_name
-~~~~~~~~~~~
+.. include:: /reference/forms/types/options/choice_name.rst.inc
 
-.. versionadded:: 2.7
-    The ``choice_name`` option was introduced in Symfony 2.7.
-
-**type**: ``string``, ``callable`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: id
-
-By default the name of each field is the id of the entity, if it can be read
-from the class metadata by an internal id reader. Otherwise the process will
-fall back to using increasing integers.
-
-choice_value
-~~~~~~~~~~~~
+In the ``EntityType``, this defaults to the ``id`` of the entity, if it can
```

Comment:
A question: does *"the id of the entity"* refer explicitly to a property called `id` or does it refer to the primary key value, whatever the property is called (and it works even for composite primary keys)?